Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:latest
+
+RUN apt-get update
+RUN apt-get install -y python3-dev python3 python3-pip curl unzip git jq
```

Comment:
Could we use an image with python installed already? Wouldn't it speed up things?

Reply:
No because awscli doesn't work on alpine builds

Reply:
Yes I looked at that. That is a base image to runs awscli directly in docker and is based off of redhat. It's no different than what I have done here

Reply:
Except that it will save the time of the `apt-get install` of python, curl and the others during the docker build time.

Reply:
The aws cli docker image has python2 installed whereas we require python3. I have just tried builds (with `no-cache`) and timed them with this dockerfile:

```
FROM amazon/aws-cli:latest

ENTRYPOINT ["/usr/bin/env"]

RUN yum update -y
RUN yum install -y python3-dev python3 python3-pip git jq

RUN alternatives --install /usr/bin/python python /usr/bin/python3 60

ADD ./requirements.txt ./
RUN pip3 install --upgrade pip
RUN pip3 install -r requirements.txt

ADD ./tests ./tests
```

Using ubuntu or awscli as a base image has very little effect on build times and actually, ubuntu is marginally faster (1:50.60 for ubuntu and 1:50.68 for awscli). In all of our other images we use either ubuntu or alpine so I think this is a sensible choice